saleor/graphql/checkout/resolvers.py

```diff
@@ -8,8 +8,10 @@ def resolve_checkout_lines():
     return queryset
 
 
-def resolve_checkouts():
+def resolve_checkouts(channel_slug):
     queryset = models.Checkout.objects.all()
+    if channel_slug:
+        queryset = queryset.filter(channel__slug=str(channel_slug))
```

Comment:
Maybe? 
In this revoler we have only `channel_slug` as `str`. `str()` is required only if we have options to pass `SimpleLesyObject` to `filter`.
```suggestion
        queryset = queryset.filter(channel__slug=channel_slug)
```